src/pages/ListView.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { List } from '../components/List';
+import { List } from '../components/List/List';
```

Comment:
We must have had a naming conflict? What kind of error message did it throw?

Reply:
I wanted to start on styling and App.css was getting long, so I create a List folder to include List.jsx and list.css!

Reply:
That makes sense! I knew there had to be a reason!